crates/rpc/rpc-api/src/validation.rs

```diff
@@ -1,9 +1,28 @@
 //! API for block submission validation.
 
+use alloy_primitives::B256;
 use alloy_rpc_types_beacon::relay::{
-    BuilderBlockValidationRequest, BuilderBlockValidationRequestV2, BuilderBlockValidationRequestV3,
+    BuilderBlockValidationRequest, BuilderBlockValidationRequestV2, SignedBidSubmissionV3,
 };
 use jsonrpsee::proc_macros::rpc;
+use serde::{Deserialize, Serialize};
+use serde_with::{serde_as, DisplayFromStr};
+
+/// A Request to validate a [`SignedBidSubmissionV3`]
+///
+/// <https://github.com/flashbots/builder/blob/7577ac81da21e760ec6693637ce2a81fe58ac9f8/eth/block-validation/api.go#L198-L202>
+#[serde_as]
+#[derive(Clone, Debug, PartialEq, Eq, Serialize, Deserialize)]
+pub struct BuilderBlockValidationRequestV3 {
```

Comment:
Why not use the one from alloy? https://github.com/alloy-rs/alloy/blob/c4f3543361db76f960bdf65cff86d5c4088fa908/crates/rpc-types-beacon/src/relay.rs#L216

Reply:
that's temporary until we have a release with https://github.com/alloy-rs/alloy/pull/1577 included